config/initializers/foreman.rb

```diff
@@ -6,14 +6,13 @@
 require_dependency 'foreman/access_permissions'
 require_dependency 'foreman/plugin'
 require_dependency 'foreman/settings'
+require_dependency 'fog_extensions'
 
 # Other internal dependencies, may be autoloaded
-require_dependency 'foreman/foreman_url_renderer'
-require_dependency 'foreman/controller'
-require_dependency 'net'
-require_dependency 'foreman/provision' if SETTINGS[:unattended]
-require_dependency 'foreman'
-require_dependency 'fog_extensions'
+require 'foreman/foreman_url_renderer'
+require 'net'
+require 'foreman/provision' if SETTINGS[:unattended]
+require 'foreman'
```

Comment:
We also require this same file in `config/application.rb#75`, should we remove it from there? I do not see a reason why we should require it there, when we do require it here...